code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -22,11 +24,16 @@
 /obj/machinery/particle_accelerator/control_box/New()
 	wires = new /datum/wires/particle_accelerator/control_box(src)
 	connected_parts = list()
+	radio = new(src)

```

Comment:
It's a proc that replaces new for many purposes

The reason behind using it is we trigger it from a subsystem, that means we have much greater control about how we space out Initializing objects, plus we can actually profile the cost of the proc much easier.